js/widgets/help.js

```diff
@@ -424,6 +424,11 @@ class HelpWidget {
         {
            rightArrow.classList.add("disabled") ;       
         }
+        if(this.index == 0)
```

Comment:
Let's use `===` here for the strict check.

Reply:
Done! Please check it. it will be my first open-source contribution if it's merged.